bonsai/examples/governance/methods/guest/Cargo.toml

```diff
@@ -17,9 +17,7 @@ k256 = { version = "=0.13.1", features = [
 ], default_features = false }
 # Directly import radium to silence warning about unused patch. See https://github.com/risc0/risc0/issues/549
 radium = "=0.7.1"
-risc0-zkvm = { path = "../../../../../risc0/zkvm", default-features = false, features = [
-  "std",
-] }
+risc0-zkvm = { version = "0.16", path = "../../../../../risc0/zkvm", default-features = false, features = ["std"] }
```

Comment:
`version =` is only needed if we plan to publish the crate

Reply:
Removed in https://github.com/risc0/risc0/pull/712/commits/dc7473d831779b05ac6c657ce7ba399c3d030a0b